core-bundle/src/Resources/contao/library/Contao/Database.php

```diff
@@ -551,7 +551,7 @@ public function getChildRecords($arrParentIds, $strTable, $blnSorting=false, $ar
 			}
 		}
 
-		return $arrReturn;
+		return array_map('\intval', $arrReturn);
```

Comment:
That change looks weird?

Reply:
It is the same thing you have added in the `getParentRecords()` method on line 574. 😊 Just making sure the method is returning integers, too.

Reply:
But this PR is all about reversing that change? :D

Reply:
It is about removing the strict checks in the DC_Table class. Returning integers in the Database methods sounds like a good idea to me, though.